CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+### Removed
+
+### Changed
+- Changed TrancriptResult field name `languageIdentifications` to `languageIdentification` and mark `languageIdentification` as optional
```

Comment:
I don't think we want these changelog entries for the finalized changelog

Reply:
Added it to trigger github workflow. Yes, ideally, we would only need for single changelog entry for feature release.